src/Components/DirectionBox/DirectionBox.jsx

```diff
@@ -17,38 +20,34 @@ const DirectionBox = ({
   const { t } = useTranslation();
   const history = useHistory();
 
+  const languages = useSelector((state) => state.language.languages);
+
   const submitDirection = () => {
     history.push(`/learn/query/${direction}`);
   };
   return (
     <div className="direction-box" onClick={submitDirection}>
       <div className="flags">
+        <Flag languageCode={foreignWordLanguage} />
         {direction === "random" ? (
-          <>
-            {languages.find((ele) => ele.name === foreignWordLanguage)?.icon}
-            <SyncAltIcon className="direction-arrow" />
-            {languages.find((ele) => ele.name === translatedWordLanguage)?.icon}
-          </>
+          <SyncAltIcon className="direction-arrow" />
         ) : direction === "backwards" ? (
-          <>
-            {languages.find((ele) => ele.name === translatedWordLanguage)?.icon}
-            <ArrowRightAltIcon className="direction-arrow" />
-            {languages.find((ele) => ele.name === foreignWordLanguage)?.icon}
-          </>
+          <ArrowRightAltIcon className="direction-arrow invert" />
         ) : (
-          <>
-            {languages.find((ele) => ele.name === foreignWordLanguage)?.icon}
-            <ArrowRightAltIcon className="direction-arrow" />
-            {languages.find((ele) => ele.name === translatedWordLanguage)?.icon}
-          </>
+          <ArrowRightAltIcon className="direction-arrow" />
```

Comment:
maybe we can show the translated-word-flag first in the second option. (e.g. German Flag - English Flag)

![Unbenannt](https://user-images.githubusercontent.com/54618409/118379127-cf468200-b5d8-11eb-9359-e6bcdd46f282.PNG)

Reply:
but then the arrow is the same as the first one.
Wrong:
![image](https://user-images.githubusercontent.com/60048565/118386816-5ff18200-b61a-11eb-8029-d50d18781a0c.png)
Correct:
![image](https://user-images.githubusercontent.com/60048565/118386823-70096180-b61a-11eb-9e18-06e7c0941ddb.png)
Old Version:
![image](https://user-images.githubusercontent.com/60048565/118386859-bd85ce80-b61a-11eb-9a66-dc5fe89d3684.png)

Reply:
Yeah, but the flags are in reading direction. It is easier to look at the flags instead of the arrows between them.

On the second direction box I just inverted the arrow

Reply:
done